src/components/Autocomplete/AutocompleteBase/index.tsx

```diff
@@ -0,0 +1,13 @@
+import {
+    Autocomplete as MaterialAutocomplete,
+    AutocompleteProps as MaterialAutocompleteProps,
+} from "@material-ui/lab";
+
+export interface AutocompleteBaseProps
+    extends MaterialAutocompleteProps<Object, boolean, boolean, boolean> {}
```

Comment:
```suggestion
export interface AutocompleteBaseProps<T>
    extends MaterialAutocompleteProps<T, boolean, boolean, boolean> {}
```
Please look into how typescript generic work. Your current implementation will make the component loses the ability to use generic type
https://www.typescriptlang.org/docs/handbook/2/generics.html#hello-world-of-generics

Reply:
My suggestion is just an example. I think it's also possible to generic the boolean variables